main.py

```diff
@@ -349,7 +359,21 @@ def build_specimin_command(project_name: str,
         jar_path_subcommand = " --jarPath" + " " + f"\"{jar_path}\""
 
     command_args = root_dir_subcommand + " " + output_dir_subcommand + " " + target_file_subcommand + " " + target_method_subcommand +  target_field_subcommand + jar_path_subcommand
-    command = "./gradlew" + " " + "run" + " " + "--args=" + f"\'{command_args}\'"
+    
+
+    command = ""
+
+    if not isWindows():
+        command = "./"
+
+    command += "gradlew" + " " + "run" + " " + "--args="
```

Comment:
On Windows, I'd expect it to be necessary to use `gradlew.bat` rather than `gradlew`. Why isn't that the case? (Maybe you're running via WSL but using the Windows file system? If so, we should document that that's the expected way to use this on Windows in the README.)

Reply:
Windows still calls `gradlew.bat` when `gradlew` is executed (it is implied). I confirmed this when I deleted the `gradlew` file on my computer, and it still works properly. I also checked that I don't have WSL enabled. Of course, it is probably better to use `gradlew.bat` since I'm not sure if earlier Windows versions act in the same way.